waku/node/waku_node.nim

```diff
@@ -225,6 +225,19 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
   if node.wakuRelay.isSubscribed(topic):
     return
 
+  proc traceHandler(topic: PubsubTopic, msg: WakuMessage) {.async, gcsafe.} =
+    notice "waku.relay received",
```

Comment:
```suggestion
    let msg_hash = topic.computeMessageHash(msg).to0xHex()
    notice "waku.relay received",
```
Nitpick I know to separate compute from log...

Reply:
Niiice, thank you! Done :))